ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -1,8 +1,24 @@
+base_images:
+  installer:
+    name: installer
+    namespace: ocp
+    tag: latest
 build_root:
   image_stream_tag:
     name: shellcheck
     namespace: ci
     tag: latest
+images:
+- dockerfile_literal: |-
+    FROM src
+    ADD openshift-install /usr/bin/openshift-install
+  from: src
+  inputs:
+    installer:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/openshift-install
+  to: installer
```

Comment:
Ideally call the resulting image something else than `installer`, installer is the name of the image that you import already.